src/content/docs/apm/agents/java-agent/getting-started/compatibility-requirements-java-agent.mdx

```diff
@@ -252,45 +252,31 @@ Before you install the Java agent, ensure your system meets these requirements:
     Compatible:
 
     * IBM JVM versions 8 for Linux
-
     * Eclipse OpenJ9 versions 8 to 13 for Linux, Windows, and macOS
-
     * OpenJDK versions 8 to 18 for Linux, Windows, and macOS
-
     * AdoptOpenJDK versions 8 to 16 for Linux, Windows, and macOS
-
     * Temurin version 17 for Linux, Windows, and macOS
-
     * Oracle Hotspot JVM versions 8 to 16 for Linux, Solaris, Windows, and macOS
-
     * Azul Zing JVM versions 8 and 11 for Linux, Windows, and macOS
-
     * Azul Zulu JVM versions 8 to 12 for Linux, Windows, and macOS
-
-    * Amazon Corretto JVM versions 8, 11, and 17 for Linux, Windows, and macOS
-
+    * Amazon Corretto JVM versions 8, 11, and 17 for Linux, Windows, and macO
     * Alibaba Dragonwell JVM versions 8 and 11 for Linux, Windows, and macOS
 
-      (Java 1.7) Compatible only with [Java agent 6.5.x](https://download.newrelic.com/newrelic/java-agent/newrelic-agent/6.5.3/newrelic-java-6.5.3.zip) \[ZIP | 16.8 MB] legacy agent:
+    (Java 1.7) Compatible only with [Java agent 6.5.x](https://download.newrelic.com/newrelic/java-agent/newrelic-agent/6.5.3/newrelic-java-6.5.3.zip) \[ZIP | 16.8 MB] legacy agent:
```

Comment:
If you look at your doc in preview, it seems odd that this line doesn't have a bullet or something. Or is this a new category?

Reply:
It's a new category, the original doc is really messed up in formatting.